uber_sante/controllers/booking_controller.py

```diff
@@ -29,25 +32,28 @@ def book():
     # TODO: connect the call to the scheduler class to try reserving the availability (line 51)
     # TODO: connect the call to the booking_service to create the booking in the Booking table (line 54)
 
-        appointment_id = request.args.get('appointment_id')
+        availability_id = request.args.get('availability_id')
         patient_id = request.args.get('patient_id')
```

Comment:
In utils > cookie_helper, define a method called "get_id_from_cookie" and return the id form the cookie; the _patient_id_ will not come from the request.args, but from the cookie. Here is an example of how to get the "logged" value from the cookie; do the same for the "id": 

![image](https://user-images.githubusercontent.com/22150653/53435931-f3055980-39c7-11e9-8e3d-0f3afa4715f9.png)